examples/gazebo_tank/differential_drive_plugin.cc

```diff
@@ -0,0 +1,110 @@
+#include <gazebo/gazebo.hh>
+#include <gazebo/physics/physics.hh>
+#include <gazebo/transport/transport.hh>
+#include <gazebo/msgs/msgs.hh>
+#include <gazebo/common/common.hh>
+
+namespace gazebo
+{
+  /// \brief A plugin to control a simple cart with differential drive.
```

Comment:
really minor thing - I don't think we generally indent within namespaces (otherwise you end up indented way across the screen before you even start!). Also this looks like it's indented with 2 spaces rather than 4

Reply:
👍